CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing to vscode-dataform-tools
+
+This document outlines the contribution guidelines for the vscode-dataform-tools extension, using a simplified Git workflow.
+
+## Git Workflow
+
+We use a streamlined branching model:
+
+* **`main` branch:** This branch always reflects the latest stable release.  Production releases are tagged from this branch.  Direct commits to `main` are generally discouraged except for hotfixes.
```

Comment:
currently, we do release after I push a version tag. So main branch in theory has code which is not released yet. We would need a CI pipeline that can hopefully streamline this process.